src/js/hooks/inboundV2/addItems/useInboundAddItemsForm.jsx

```diff
@@ -485,9 +497,9 @@ const useInboundAddItemsForm = ({
     }
   };
 
-  const setLineItems = (response, startIndex) => {
+  const setLineItems = (response, startIndex, showOnlyImportedItems) => {
     const { data } = response.data;
-    const lineItemsData = data.length === 0 && getValues('values.lineItems').length === 0
+    const lineItemsData = !data.length && getValues('values.lineItems').length === 0
```

Comment:
You removed comparing to 0 for the first part of the condition, but you left exactly the same situation on the right side :smile: